### PR TITLE
chore(CHANGELOG): update to 5.0.0-alpha.0

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,69 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
 * `FIX`: keep selection of a replaced element ([#2306](https://github.com/camunda/camunda-modeler/issues/2306))
 * `DEPS`: update to `dmn-js@11.1.2`
 
+### Forms
+
+* `FEAT`: split Camunda Platform and Camunda Cloud forms ([#2650](https://github.com/camunda/camunda-modeler/issues/2650))
+* `FEAT`: allow to deploy Camunda Platform forms ([#2498](https://github.com/camunda/camunda-modeler/issues/2498))
+* `FEAT`: allow to drag and drop forms ([#2490](https://github.com/camunda/camunda-modeler/issues/2490))
+* `FIX`: properly set exporter metadata ([#2540](https://github.com/camunda/camunda-modeler/issues/2540))
+* `FIX`: enable "Select all" in the properties panel ([#2411](https://github.com/camunda/camunda-modeler/issues/2411))
```
Comment (philippfromme) [inline CHANGELOG.md:62]:
```suggestion
* `FIX`: enable "Select all" in properties panel ([#2411](https://github.com/camunda/camunda-modeler/issues/2411))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,11 +6,69 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FEAT`: make file permissions for Linux more strict ([#2439](https://github.com/camunda/camunda-modeler/issues/2439))
+* `FEAT`: improve welcome tab UI ([#2470](https://github.com/camunda/camunda-modeler/issues/2470), [#2479](https://github.com/camunda/camunda-modeler/issues/2479))
+* `FIX`: restrict height of log ([#2258](https://github.com/camunda/camunda-modeler/issues/2258))
+* `FIX`: require at least one item of system information to be checked ([#2414](https://github.com/camunda/camunda-modeler/issues/2414))
+* `DEPS`: update to `electron@12.1.2`
+
+### BPMN
+
+* `FEAT`: support Camunda Cloud 1.4 ([[#2524](https://github.com/camunda/camunda-modeler/issues/2524)]), ([[#2641](https://github.com/camunda/camunda-modeler/issues/2641)])
+* `FEAT`: UX and technical re-write of the Properties Panel for Camunda Platform diagrams  ([#2663](https://github.com/camunda/camunda-modeler/issues/2663))
```
Comment (philippfromme) [inline CHANGELOG.md:36]:
```suggestion
* `FEAT`: UX and technical re-write of the properties panel for Camunda Platform diagrams  ([#2663](https://github.com/camunda/camunda-modeler/issues/2663))
```

**Review by philippfromme — COMMENTED** [review]
Great stuff. In general, we should stick to lowercase except for product names and either stick to not using articles (e.g. _fix properties panel_) or decide to always use them (e.g. _fix the properties panel_). Sorry for being petty about this. 🤡

**Comment by MaxTru** [comment]
We could also mention https://github.com/camunda/camunda-modeler/issues/2440 and https://github.com/camunda/camunda-modeler/issues/1240

**Comment by barmac** [comment]
Updated.

**Review by philippfromme — APPROVED** [review]
Go! 🏁

**Pull request merged**